skills/lark-contact/references/lark-contact-search-user.md

```
@@ -1,39 +1,185 @@
+
+默认只请求一页。返回 `has_more=true` 时,推荐先**收敛 filter**;确实要拉多页再启用自动翻页。
+
+| 传的 flag | 模式 | 行为 |
+|---|---|---|
+| (都不传) | 单页 | 返 1 页;`has_more=true` 时 stderr 提示收敛 filter 或用 `--page-all` |
+| `--page-all` | 自动(上限 40) | 自动循环直到 `has_more=false` 或达到 40 页 |
+| `--page-limit 5` | 自动(上限 5) | 隐式启用 auto,最多 5 页 |
+| `--page-all --page-limit 5` | 自动(上限 5) | 跟只传 `--page-limit 5` 等价 |
+
+到达 `--page-limit` 仍有 `has_more` 时,stderr 会出:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)